CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change Log
 All notable changes to HiOp are documented in this file.
 
+## Version 0.7.1: Miscellaneous fixes to build system
+This minor release fixes a couple of issues found in the build system after the major release 0.7 of HiOp. 
+
```

Comment:
I think this is sufficient documentation, although I wasn't sure if we should be referencing closed issues in cases like this.